demos/notebooks/demo_pipeline.ipynb

```diff
@@ -195,15 +269,31 @@
     "            'min_cnn_thr': cnn_thr,\n",
     "            'cnn_lowest': cnn_lowest}\n",
     "\n",
-    "opts = params.CNMFParams(params_dict=opts_dict)"
+    "parameters = params.CNMFParams(params_dict=parameter_dict)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# check some of your parameter clusters, if you want\n",
+    "parameters.data"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Setup a cluster\n",
-    "To enable parallel processing a (local) cluster needs to be set up. This is done with a cell below. The variable `backend` determines the type of cluster used. The default value `'local'` uses the multiprocessing package. The `ipyparallel` option is also available. More information on these choices can be found [here](https://github.com/flatironinstitute/CaImAn/blob/master/CLUSTER.md). The resulting variable `dview` expresses the cluster option. If you use `dview=dview` in the downstream analysis then parallel processing will be used. If you use `dview=None` then no parallel processing will be employed."
+    "## Setting up cluster\n",
+    "To enable parallel processing a (local) cluster needs to be set up. This is done with the `setup_cluster()` function  below, which sets up a pool of processors using the multiprocessing package. \n",
+    "\n",
+    "The `backend` parameter determines the type of cluster used. The default value `'local'` uses the multiprocessing package. The `ipyparallel` option is also available. More information on these choices can be found [here](https://github.com/flatironinstitute/CaImAn/blob/master/docs/CLUSTER.md). You can set the number of processes (cpu cores) to use with the `n_processes` variable: the default value `None` will lead to the function selecting one less than the total number available. \n",
```

Comment:
We should steer people away from using the name "local".
We should also change the default to the non-aliased value (which should be harmless).